src/components/MyDetails.vue

```diff
@@ -1,90 +1,128 @@
 <script setup >
 import { ref } from 'vue'
 
-const props = defineProps({
+const componentProps = defineProps({
   name: String,
-  email: String
+  email: String,
+  avatar: String
```

Comment:
Here again, let's call it profilePicture. Also true for the labels if there are any on the UI. On the label, we should say "Profile picture"